frontend/src/pages/LoginRegister/RegisterPage.vue

```diff
@@ -58,21 +65,23 @@ export default defineComponent({
   },
   methods: {
     handleRegister() {
-      if (this.password !== this.confirmPassword) {
-        // TODO: show some error here.
-        return;
-      }
-      this.store.dispatch("register", {
-        fullName: this.fullName,
-        username: this.username,
-        password: this.password,
-      }).then(() => this.$router.push(HOME_ROUTE))
-        // TODO: Display some form of user feedback upon registration failure.
-        .catch((err) => console.log(err, "Failed to register"));
+      this.store
+        .dispatch("register", {
+          fullName: this.fullName,
+          username: this.username,
+          password: this.password,
+        })
+        .then(() => this.$router.push(HOME_ROUTE))
+        // TODO: maybe map some error messages to more user facing messages
+        // e.g HTTP response at 400 or 500 => Not connected to server or smth
```

Comment:
I guess we could, but then we would need to handle it at some API abstraction. Don't think we have the time for that. Let's just assume it should never happen (with our deployment setup)?